silx/gui/plot/actions/control.py

```diff
@@ -632,3 +632,73 @@ def _modeChanged(self, source):
 
     def _actionTriggered(self, checked=False):
         self.plot._eventHandler.validate()
+
+
+class OpenGlAction(PlotAction):
```

Comment:
Best renamed to `OpenGLAction` to follow naming convention.

Reply:
This one is fixed